### PR TITLE
chore(internal/kokoro): use 1.15 for regen

### PR DESCRIPTION
**Pull request by codyoss**

### CODE REVIEW

File: internal/kokoro/synth/synth.sh [hunk 1]
```diff
@@ -7,6 +7,13 @@
 set -e -x
 
 cd $(dirname $0)/../../..
+
+# Install Go 1.15
+curl -o /tmp/go.tgz https://dl.google.com/go/go1.15.1.linux-amd64.tar.gz && \
+    tar -C /usr/local -xzf /tmp/go.tgz && \
```
Comment (tbpg) [inline internal/kokoro/synth/synth.sh:13]: Is Go already installed at `/usr/local`? If so, we need to delete it first.

File: internal/kokoro/synth/synth.sh [hunk 2]
```diff
@@ -7,6 +7,13 @@
 set -e -x
 
 cd $(dirname $0)/../../..
+
+# Install Go 1.15
```
Comment (tbpg) [inline internal/kokoro/synth/synth.sh:11]: Seems like this should be updating some container image, rather than reinstalling Go?

**Review by tbpg — CHANGES_REQUESTED** [review]

File: internal/kokoro/synth/synth.sh (hunk 1 above)
Comment (codyoss) [reply internal/kokoro/synth/synth.sh]: Done.

File: internal/kokoro/synth/synth.sh (hunk 2 above)
Comment (codyoss) [reply internal/kokoro/synth/synth.sh:11]: This is not run in a container today.

**Review by tbpg — APPROVED** [review]

**Pull request merged**